Directory.Build.targets

```diff
@@ -20,6 +20,11 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsTestProject)' != 'true' and '$(IsSourceProjectPackable)' == 'true'">
+    <!-- IsSourceProjectPackable is used in the CI packing leg to enable IsPackable only on source projects. -->
+    <IsPackable>true</IsPackable>
+  </PropertyGroup>
```

Comment:
You need to set this before the packaging.targets import above that conditions on `IsPackable=true` happens. Otherwise none of the packaging.targets (including the switch to enable package validation) will take effect.